doc/conf/YAML.md

```diff
@@ -0,0 +1,45 @@
+# Yaml - Configuration file
+
+To use read from Yaml file option it must has be installed.
+It it was not and, for some reason you don't want or don't can install it, you still can make it at regular php file, see more [here](./PHP.md).
```

Comment:
**If** it was not installed or for some other reason you don't want it or can't install it, you still can make it at regular php configuration file, see more [here](./PHP.md).